ONSdigital/ras-collection-instrument
# Add collection instrument form type check eq and seft

# What and why?
This PR validates if a collection instrument can be uploaded according to it's form_type. 
# How to test?
Test with this PR deploy as well https://github.com/ONSdigital/response-operations-ui/pull/844
Using this PR create a EQ_AND_SEFT survey in rops, upload an eQ form_type (i.e 0001) then try to upload a SEFT using the same form type (a file named 064_201803_0001 would do it), it should return a warning. Repeat the process but the other way around Upload a SEFT first then an eQ. This is survey wide, so try across CE as well (i.e create a CE and upload a eQ, then in a different CE try to add a SEFT with the same form_type)
# Trello
